pip_install_privates/install.py

```diff
@@ -104,8 +104,12 @@ def install():
     parser.add_argument('req_file', help='path to the requirements file to install')
     args = parser.parse_args()
 
+    token = args.token
```

Comment:
why not specify a default?
```
    parser.add_argument('--token', '-t', help='Your Personal Access Token for private GitHub repositories')
```
```
    parser.add_argument('--token', '-t', help='Your Personal Access Token for private GitHub repositories', default=os.environ.get('GITHUB_TOKEN'))
```

Reply:
Got caught being lazy :smile: I've had some annoying experiences with defaults in argparser, but will do :slightly_smiling_face: